boards/arc/axs/doc/index.rst

```diff
@@ -0,0 +1,206 @@
+.. _axs:
+
+DesignWare(R) ARC(R) AXS Software Development platform
+######################################################
+
+Overview
+********
+
+The DesignWare(R) ARC(R) Software Development Platforms are complete, standalone
+platforms that enable software development, debugging and profiling, which consists
+of a mainboard with a CPU card. Synopsys currently has two Software Development
+Platforms available: AXS101 and AXS103
+
+.. image:: ./axs.jpg
+   :width: 442px
+   :align: center
+   :alt: DesignWare(R) ARC(R) AXS Software Development platform (synopsys.com)
+
+For details about the board, see: `DesignWare ARC Software Development Platforms
+<https://www.synopsys.com/dw/ipdir.php?ds=arc-software-development-platform>`__
+
+
+Hardware
+********
+
+For hardware feature details, refer to :
+
+`DesignWare ARC AXC001 CPU Card Datasheet
+<https://www.synopsys.com/dw/doc.php/ds/cc/arc_axc001_cpu_card.pdf>`__
+
+`DesignWare ARC AXC003 CPU Card Datasheet
+<https://www.synopsys.com/dw/doc.php/ds/cc/arc_axc003_cpu_card.pdf>`__
+
+`DesignWare ARC AXS101 Software Development Platform Datasheet
+<https://www.synopsys.com/dw/doc.php/ds/cc/arc_axs101_sdp.pdf>`__
+
+`DesignWare ARC AXS103 Software Development Platform Datasheet
+<https://www.synopsys.com/dw/doc.php/ds/cc/arc_axs103_sdp.pdf>`__
+
+
+Programming and Debugging
+*************************
+
+Required Hardware and Software
+==============================
+
+To use Zephyr RTOS applications on the AXS Software Development platform,
+a few additional pieces of hardware are required.
+
+* A micro USB cable provides USB-JTAG debug and USB-UART communication
+  to the board
+
+* A universal switching power adaptor (110-240V
+  AC to 12V DC), provided in the package, provides power to the board.
+
+* :ref:`The Zephyr SDK <zephyr_sdk>`
+
+* Terminal emulator software for use with the USB-UART. Suggestion:
+  `Putty Website`_.
+
+* (optional) A collection of Pmods, Arduino modules, or Mikro modules.
+  See `Digilent Pmod Modules`_ or develop your custom interfaces to attach
+  to the Pmod connector.
+
+Set up the AXS Software Development platform
+============================================
+
+To run Zephyr application on AXS Software Development platform,
+you need to setup the board correctly.
+
+* Connect the digilent usb cable from your host to the board.
+
+* Connect the 12V DC power supply to your board
+
+Set up Zephyr Software
```

Comment:
If there's no content for this section, then delete the "Set up Zephyr Software" heading (and underline)